src/controllers/auth.js

```diff
@@ -0,0 +1,45 @@
+import { config } from 'dotenv';
+import userServices from '../services/user';
+import successRes from '../utils/successResponse';
+import ErrorResponse from '../utils/errorResponse';
+import encryption from '../helpers/encryption';
+
+config();
+
+const duration = process.env.TOKEN_EXPIRE;
+
+const { findUser } = userServices;
+const { signToken, checkPassword } = encryption;
+
+class Auth {
+  static async login(req, res, next) {
+    try {
+      const { email, password } = req.body;
+      const foundUser = await findUser({ email });
+      if (!foundUser) return ErrorResponse(res, 404, 'User Not found');
```

Comment:
Don't return **user not found**.
For security purposes please just say **Incorrect credentials**